src/renderer/components/recording-page/recording-page.test.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable no-console */
+/* eslint-disable jest/prefer-expect-assertions */
+
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+import { noOp } from '../../env-and-consts';
+import mediaService from '../../services/media';
+
+import { RecordingPage } from './index';
+
+jest.mock('../../services/media', () => ({
+  switchOnAudioInput: jest.fn(noOp),
```

Comment:
If you leave this out, you can specify the behavior separately. See #85

Reply:
I tried just using jest.mock('../../services/media'), but it wouldn't accept the mock unless I added in a function for the mock to use.  Should i set it so that the jest.mock call passes in an empty function initially, and then specify the behavior for each method later?

Reply:
It depends. Show me the exact error (after you rebase the branch).

If default mock doesn't work, then you can still do `switchOnAudioInput: jest.fn(),` and specify the behavior later.